### ISSUE TITLE
Pod mopub-ios-sdk cannot be prebuilt

### ISSUE THREAD

**Issue by iKleMiX** [Subito-it/PodBuilder#5]
If you add `pod 'mopub-ios-sdk', '5.7.1'` to Frameworks/Podfile it will fail to build:
```% pod_builder build_all
Updating local specs repositories
...
Analyzing dependencies
Analyzing dependencies
Initialized empty Git repository in /private/tmp/pod_builder_1575564641353/.git/
Analyzing dependencies
Downloading dependencies
Installing AWSCore (2.6.28)
Installing AWSPinpoint (2.6.28)
Installing FBSDKCoreKit (5.12.0)
Installing GoogleUtilities (6.3.2)
Installing MDFInternationalization (2.0.0)
Installing MDFTextAccessibility (2.0.0)
Installing MaterialComponents (94.5.0)
Installing SSZipArchive (2.2.2)
Installing mopub-ios-sdk (5.7.1)
Installing nanopb (0.3.9011)
Generating Pods project
Skipping User Project Integration
Building frameworks
Built 20 frameworks
Copying 22 frameworks to `Rome`
Pod installation complete! There are 13 dependencies from the Podfile and 10 total pods installed.
Unable to detect item for framework MPUBMoatMobileAppKit.framework. Please open a bug report!
/Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/install.rb:108:in `block in add_framework_plist_info'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/install.rb:83:in `glob'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/install.rb:83:in `add_framework_plist_info'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/install.rb:39:in `podfile'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/command/build.rb:77:in `block in call'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/command/build.rb:73:in `each'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/command/build.rb:73:in `call'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/lib/pod_builder/command/build_all.rb:11:in `call'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/exe/pod_builder:320:in `block in parse_commandline'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/exe/pod_builder:319:in `each'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/exe/pod_builder:319:in `parse_commandline'
        /Library/Ruby/Gems/2.6.0/gems/pod-builder-0.9.1/exe/pod_builder:337:in `<top (required)>'
        /usr/local/bin/pod_builder:23:in `load'
        /usr/local/bin/pod_builder:23:in `<main>'


Command failed!
```

**Comment by tcamin** [comment]
Hi, thanks for the feedback as indeed there was an issue prebuilding mopub-ios-sdk.

I released version 0.9.4 today which ships a significant number of improvements, could you try with this version again. I tried prebuilding it on a sample project and it is working now.

**Comment by iKleMiX** [comment]
This now builds correctly. Thank you.
